ONSdigital/eq-survey-runner
# Added Codacy Python test coverage upload to Travis

### What is the context of this PR?
Added an upload of the Python coverage file to codacy and cleaned-up the travis config a bit.

### How to review 
- Check if Codacy shows coverage for the PR.
- Check the Codacy badge works in the README.md
- Check that the Travis build works correctly (possibly compare output to master to check for differences)
